packages/data-point/README.md

```diff
@@ -1462,24 +1462,30 @@ To customize type checking you may use a [Reducer](#reducers). If the reducer th
 </details>
 
 
-#### <a name="transform-entity">Transform Entity</a>
+#### <a name="reducer-entity">Reducer Entity</a>
```

Comment:
I noticed that this line is in another section of the readme:

`for all entites except transform`

do you mind replacing that sentence with:

`for all entities except [Reducer](#reducer-entity)`
?